tests/wgsl/math.slang

```diff
@@ -0,0 +1,285 @@
+//TEST:SIMPLE(filecheck=WGSL): -stage compute -entry computeMain -target wgsl
```

Comment:
Since we only have `WGSL:` comments in the test, I guess we don't need the other test lines here.

Reply:
I removed other targets.